blog/2021-08-12-sensor_temperature_conversion.md

```diff
@@ -0,0 +1,32 @@
+---
+author: Erik Montnémery
+authorURL: https://github.com/emontnemery
+title: Temperature conversions moving to SensorEntity
+---
+
+Temperature unit conversions are moving from the state machine to the SensorEntity base
```

Comment:
They are moving from the entity base class to the sensor entity base class, not from the state machine ?